lib/ansible/config/base.yml

```diff
@@ -1255,7 +1255,7 @@ INVENTORY_ENABLED:
   type: list
 INVENTORY_IGNORE_EXTS:
   name: Inventory ignore extensions
-  default: eval(BLACKLIST_EXTS + ( '~', '.orig', '.ini', '.cfg', '.retry'))
+  default: "{{(BLACKLIST_EXTS + ( '~', '.orig', '.ini', '.cfg', '.retry'))|list}}"
```

Comment:
Note: piping to list probably isn't needed (it's what literal_eval will do)

Reply:
would get tuple